content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,301 +1,199 @@
-
-<details><summary>If you deploy GraphQL servers to AWS Lambda</summary>
-
-This example uses REST, but the same principles apply to a GraphQL server.<br /><br />The main difference is that you typically only need a single function to serve a
-GraphQL API. In that function, set the `context.callbackWaitsForEmptyEventLoop` of the [AWS Lambda Context Object](https://docs.aws.amazon.com/lambda/latest/dg/nodejs-context.html) to `false` as follows:
-
-```js
-exports.server = (event, context, cb) => {
-  // Set to false to send the response right away when
-  // the callback executes. Don't wait for the Node.js
-  // event loop to be empty.
-  context.callbackWaitsForEmptyEventLoop = false
-
-  return lambda.graphqlHandler(event, context, cb)
-}
-```
-
-</details>
+This guide explains how to avoid common issues when deploying a [Serverless Framework](https://www.serverless.com/framework/) project with Prisma. For a guide on how to get started with Serverless Framework, we recommend [Setting Up Serverless Framework With AWS](https://www.serverless.com/framework/docs/getting-started).
 
 </TopBlock>
 
 ## Prerequisites
 
-- A hosted PostgreSQL database and a URL from which your application can access it, such as `postgresql://username:password@your_postgres_db.cloud.com/db_identifier` (you can use Supabase, which offers a [free plan](https://dev.to/prisma/set-up-a-free-postgresql-database-on-supabase-to-use-with-prisma-3pk6)).
-- An [AWS](https://aws.amazon.com/) account and a corresponding [access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) for programmatic access.
-- A [Serverless Framework CLI](https://www.serverless.com/framework/docs/getting-started/) installed. See the note below about [Serverless Framework CLI commands](#serverless-framework-cli-commands).
-- PostgreSQL CLI `psql` installed.
-- Node.js installed.
-
-### Serverless Framework CLI commands
-
-The exact Serverless Framework commands that you must use depend on the way that you install the Serverless Framework CLI.
-
-- If you install the Serverless Framework CLI globally, then you can use the commands exactly as shown in this guide.
-
-- If you install the Serverless Framework CLI in your project, then you must prepend `npx` to the commands in this guide. For example:
-
-  ```terminal wrap
-  npx serverless config credentials --provider aws --key AWS_ACCESS_KEY_ID  --secret AWS_SECRET_ACCESS_KEY
-  ```
-
-## Prisma workflow
+- An existing Serverless Framework project with Prisma.
 
-In this guide, you do the following:
+## Things to know
```

Comment:
Maybe remove this headline and just have the current `###` as `##` directly? WOuld also make the actual problems visible in the right hand ToC.